### ISSUE TITLE
N2630 is not compatible with Agnus 8371

### ISSUE THREAD

**Issue by coriolis1986** [jasonsbeer/Amiga-N2630#20]
Hi all.

Just tried to run my A2000 with N2630 and Agnus 8371. It crashes with an old firmware. I reflashed U600-U602 to latest versions and my Amiga just refuse to start anymore.

**Comment by LIV2** [comment]
Hi @coriolis1986 

Did you previously have an 8372 fitted and did it work with the N2630 in this machine?

If you did swap from an 8372 to 8371 did you also change J101 to match this and close J500?

What version of Kickstart are you using and do you have the 68000 installed in the motherboard?

**Comment by coriolis1986** [comment]
The original configuration of this A2000 has an 8371. I replaced it for a 8372, which I took out from Amiga 500. And N2630 works perfectly after this.

I have a Kickstart switcher (1.3, 2.04, 3.1, 3.2.1) and any of them works with an accelerator. Also the 68000 installed in the motherboard.

Both of the 8371 and 8372 works with 68000.

So only when the Agnus is changed, N2630 started or stopped to work.

I'll try to look to the jumpers on motherboard (ver 4.4). May be some of them will change the situation.
